src/imaevm.h

```diff
@@ -27,8 +27,8 @@
 #include <openssl/rsa.h>
 #include <openssl/opensslconf.h>
 
-#if !defined(OPENSSL_NO_ENGINE) && !defined(OPENSSL_NO_DYNAMIC_ENGINE)
-# include <openssl/engine.h>
+#if CONFIG_IMA_EVM_ENGINE
```

Comment:
You cannot do this in a public header file that other apps may #include. They don't know about this CONFG_IMA_EVM_ENGINE.

Here is what seems to be necessary now with rawhide:
`+#if !defined(OPENSSL_NO_ENGINE) && !defined(OPENSSL_NO_DYNAMIC_ENGINE) && !defined(OPENSSL_NO_STATIC_ENGINE)`